x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/servicenow/application_required_callout.test.tsx

```diff
@@ -9,22 +9,32 @@ import React from 'react';
 import { render, screen } from '@testing-library/react';
 import { ApplicationRequiredCallout } from './application_required_callout';
 
+const appId = 'test';
+
 describe('ApplicationRequiredCallout', () => {
   test('it renders the callout', () => {
-    render(<ApplicationRequiredCallout />);
+    render(<ApplicationRequiredCallout appId={appId} />);
```

Comment:
If this ids is something which will never change, then I think the better approach to avoid passing this as props and just use a constants

Reply:
The reason I pass the id as a pros is that the component is used by both the ITSM & SecOps so it is different for each connector. Do you want me to do something different?

Reply:
Thank you for the explanation. It seems to be right.